plugins/woocommerce-admin/client/customize-store/design-with-ai/state-machine.tsx

```diff
@@ -62,8 +63,12 @@ export const designWithAiStateMachineDefinition = createMachine(
 						},
 					},
 					postBusinessInfoDescription: {
-						always: {
-							target: '#lookAndFeel',
+						invoke: {
+							src: 'getLookAndTone',
+							onDone: {
+								actions: [ 'assignLookAndTone' ],
```

Comment:
We need to handle errors here. I think we can just log the errors and continue to Look and Tone steps. cc @adrianduffell  for thoughts.

Reply:
This sounds good 👍

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/39979/commits/097b512a7905ac16810af7464536fa67c8690375

For now, I've just added a logger function (it doesn't track anything yet).